src/Controller/Component/PaginatorComponent.php

```diff
@@ -222,6 +222,7 @@ public function paginate($object, array $settings = [])
             'sortDefault' => $sortDefault,
             'directionDefault' => $directionDefault,
             'scope' => $options['scope'],
+            'totalOrder' => $order
```

Comment:
Total sounds like like a sum, which this isn't. Could something like 'completeOrder' or 'fullOrder' work?